docs/developer_guide/contributing.md

```diff
@@ -0,0 +1,100 @@
+# Contributing
+
+Everybody is welcome to contribute, and we value everybody’s contribution. Code is thus not the only
+way to help the community. Answering questions, helping others, reaching out and improving the
+documentation are immensely valuable contributions as well.
+
+It also helps us if you spread the word: reference the library from blog posts on the awesome
+projects it made possible, shout out on Twitter every time it has helped you, or simply star the
+repo to say "thank you".
+
+Check out the official [ludwig docs](https://ludwig-ai.github.io/ludwig-docs/) to get oriented
+around the codebase, and join the community!
+
+## Open Issues
+
+Issues are listed at: <https://github.com/ludwig-ai/ludwig/issues>
+
+If you would like to work on any of them, make sure it is not already assigned to someone else.
+
+You can self-assign it by commenting on the Issue page with one of the keywords: `#take` or
+`#self-assign`.
+
+Work on your self-assigned issue and eventually create a Pull Request.
+
+## Creating Pull Requests
+
+1. Fork the [repository](https://github.com/ludwig-ai/ludwig) by clicking on the "Fork" button on
+   the repository's page. This creates a copy of the code under your GitHub user account.
+
+1. Clone your fork to your local disk, and add the base repository as a remote:
+
+   ```bash
+   git clone git@github.com:<your Github handle>/ludwig.git
+   cd ludwig
+   git remote add upstream https://github.com/ludwig-ai/ludwig.git
+   ```
+
+1. Create a new branch to hold your development changes:
+
```

Comment:
I know that things update correctly when the README is rendered, but are we able to increment these to 2., 3. etc just for better readability?

Reply:
One more idea for more principled development here is to create a branch from the UI. This creates a link between a branch name and the issue name, which is kind of nice. 

<img width="1581" alt="Screen Shot 2022-09-06 at 11 30 11 AM" src="https://user-images.githubusercontent.com/106701836/188711956-21ab955d-2ee1-48e1-b0d8-4890fadb7e95.png">

Reply:
> I know that things update correctly when the README is rendered, but are we able to increment these to 2., 3. etc just for better readability?

done.

> One more idea for more principled development here is to create a branch from the UI (bottom right). This creates a link between a branch name and the issue name, which is kind of nice.

I'm not familiar with that workflow, would you be able to add this to doc? Maybe in a separate PR.

Reply:
Yup, I can add that as a follow-up PR.